Libs/DICOM/Widgets/ctkDICOMAppWidget.cpp

```diff
@@ -55,7 +55,9 @@
 #include "ctkDICOMThumbnailGenerator.h"
 #include "ctkThumbnailLabel.h"
 #include "ctkDICOMQueryResultsTabWidget.h"
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.